src/static.jl

```diff
@@ -6,6 +6,10 @@ Use `Static(N)` instead of `Val(N)` when you want it to behave like a number.
 struct Static{N} <: Integer
     Static{N}() where {N} = new{N::Int}()
 end
+
+const Zero = Static{0}()
+const One = Static{1}()
```

Comment:
I said `const Zero = Static{0}()`, but what I meant was `const Zero = Static{0}`, so that we can use `::Zero` or `::One` in method signatures.